.travis.yml

```diff
@@ -1,9 +1,19 @@
 language: java
 
+os:
+  - linux
+  - osx
+
 jdk:
+  - oraclejdk11
   - openjdk8
   - openjdk11
 
+matrix:
+  exclude:
+    - os: osx
+      jdk: openjdk8
```

Comment:
Why does this combination have to be excluded?

Reply:
This combination is not supported. See [here](https://travis-ci.org/junit-pioneer/junit-pioneer/jobs/652617824)
`Expected feature release number in range of 9 to 15, but got: 8`
They only support openJDK 9 to 15 for osx.